wp-content/civi-extensions/goonjcustom/Civi/GoonjUniqueUserValidation.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Civi;
+
+use CRM_Goonjcustom_ExtensionUtil as E;
+use Civi\Afform\Event\AfformValidateEvent;
+use Civi\Core\Service\AutoSubscriber;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+class GoonjUniqueUserValidation extends AutoSubscriber implements EventSubscriberInterface
+{
+
+    /**
+     * @return array
+     */
+    public static function getSubscribedEvents()
+    {
+        return [
+        'civi.afform.validate' => ['onAfformValidate'],
+        ];
+    }
+
+    public static function onAfformValidate(AfformValidateEvent $event)
+    {
+        $formTitle = $event->getAfform()['title'] ?? '';
+
+        if ($formTitle !== 'Volunteer Registration') {
```

Comment:
Instead of comparing with the form title (which can be changed by user) can we compare the form name (`afformIndividualRegistration1` in the screenshot)
<img width="1279" alt="image" src="https://github.com/user-attachments/assets/d79e9627-ecc4-4ffe-aac4-82f5dc78f609">

Reply:
Done